kratos.gid/apps/Fluid/write/writeProjectParameters.tcl

```diff
@@ -3,10 +3,10 @@ proc ::Fluid::write::getParametersDict { } {
     set projectParametersDict [dict create]
 
     # Problem data
-    dict set projectParametersDict problem_data [write::GetDefaultProblemDataDict $Fluid::app_id]

```

Comment:
@jginternational everything works after this change but please confirm me that it is OK.

Reply:
This was set to solve the inheritance problem of calling to common functions from the inherited app.

In this case imagine the FSI

the function apps::getActiveAppId will return FSI but you want to execute the function telling the system to search in the Fluid part of the tree, so this is why $Fluid::app_id must be there

Reply:
OK! I'll revert it. Thanks for the explanation.